README.md

```diff
@@ -53,244 +65,68 @@ coastal_nomangrove_minus_mangrove__rp_100__rcp_baseline__epoch_2010__conf_None.t
 ```
 
 And in `tileserver/vector/`, mbtiles files like these:
-
 ```
 airport_runways.mbtiles
 airport_terminals.mbtiles
 buildings_commercial.mbtiles
 buildings_industrial.mbtiles
 ```
 
-### Docker Development Environment
```

Comment:
Super helpful to have these in actual files with example values 👍🏻 

Is there anything that should be changed from `envs/dev-example` when copying to `envs/dev`?

Reply:
Just trying things out - docker compose complains that `.pgadmin.env` is missing - could drop that service from the dev compose? Or add an example file to `envs/dev-example`

Reply:
Suggest reviewing the "profiles" feature of docker compose ([docs](https://docs.docker.com/compose/profiles/)) - this could be used to assign multiple profiles to each service, e.g. one profile could be `frontend-dev` which would spin up all the necessary services such as `backend`, `db`, `tiles-db` etc

Reply:
I had a look at the profiles, but I don't think I'll define any as the set of services you'll want up depend on the job you're doing, and I think there are potentially quite a few different jobs. I think I'll leave it up to the developers discretion which services to start.